app/assets/templates/static_pages/index.html

```diff
@@ -0,0 +1,69 @@
+<div class="container">
+  <div class="row">
+    <h1> {{ message }} </h1>
+  </div>
+</div>
+
+<form>
```

Comment:
The form is used to create book. However, there is a search function in the title and isbn fields which helps user to input data faster by getting details of the book using google api. 
The search function detects "ENTER" key, while ng-submit detects "ENTER" key too. Thats why ng-click and ng-keypress were used to separate two functions.